src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,42 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const orders = await knex('orders AS o')
+      // .select('orders.id as id', 'status', 'quantity', 'name')
+      .select(
+        'o.id as orderId',
+        'o.status as orderStatus',
+        'o.user_id',
```

Comment:
do you need user here?

Reply:
to have access to user info as we have them also in page, however we have them .Maybe remove it.